src/common/cloudinary.ts

```diff
@@ -108,8 +108,12 @@ export const uploadPostFile = (
   preset: PostPreset,
 ) => uploadFile(name, preset, stream);
 
-export const mapCloudinaryUrl = (url?: string): string | undefined =>
-  url?.replace(
+export function mapCloudinaryUrl(url: string): string;
+export function mapCloudinaryUrl(url: undefined): undefined;
+export function mapCloudinaryUrl(url?: string): string | undefined;
+export function mapCloudinaryUrl(url?: string): string | undefined {
```

Comment:
Is this intentional/correct?
If so why is it needed?

Reply:
Surely you can do the last one alone and early return if no value is given?
If you accept string|undefined?

Reply:
But type wise I have to return string | undefined which violates ts strict. That it is how it was before I added the new use case

Reply:
Ah I see, not the biggest fan of overloads in JS world, but guess it's ok if it works 😂